### PR TITLE
[SM64] Only throw empty gfx exception in actor exports

### PR DESCRIPTION
**Pull request by Lilaa3**
Oh lila LOVESSS breaking main RIGHT SCUT?

### CODE REVIEW

File: fast64_internal/sm64/sm64_geolayout_writer.py [hunk 1]
```diff
@@ -978,7 +977,7 @@ def exportGeolayoutObjectBinary(
     camera,
 ):
     geolayoutGraph, fModel = convertObjectToGeolayout(
-        obj, convertTransformMatrix, camera, obj.name, None, None, DLFormat.Static, True
+        obj, convertTransformMatrix, True, obj.name, None, None, DLFormat.Static, True
```
Comment (jesusyoshi54) [inline fast64_internal/sm64/sm64_geolayout_writer.py:980]: You should remove the "camera" argument from the functions if it is not used.

**Review by jesusyoshi54 — CHANGES_REQUESTED** [review]
I tested this and it works and the code is simple enough.

On an somewhat unrelated note, the `exportGeolayoutArmature` have camera arguments with some esoteric uses that are lost to time and never used. This should be cleaned up at some point.

File: fast64_internal/sm64/sm64_geolayout_writer.py (hunk 1 above)
Comment (Lilaa3) [reply fast64_internal/sm64/sm64_geolayout_writer.py:980]: that is what I did?

Comment (jesusyoshi54) [reply fast64_internal/sm64/sm64_geolayout_writer.py:980]: yeah remove the `camera` from the function `exportGeolayoutObjectBinary` and others that do not actually use the argument.

**Comment by jesusyoshi54** [comment]
When you call `exportGeolayoutObjectBinaryBank0` it still has the `camera` argument present. Same with `exportGeolayoutObjectBinary` and `exportGeolayoutObjectC`
```python
Traceback (most recent call last):
  File "C:\Users\scut\Desktop\fast64-main\addons\fast64\fast64_internal\sm64\sm64_objects.py", line 1854, in execute
    self.execute_gfx(props, context, obj, index)
  File "C:\Users\scut\Desktop\fast64-main\addons\fast64\fast64_internal\sm64\sm64_objects.py", line 1843, in execute_gfx
    raise Exception(e)
Exception: Error: exportGeolayoutObjectC() takes 13 positional arguments but 14 were given
```

**Comment by Lilaa3** [comment]
sorry about that I will pay more attention next time

**Review by jesusyoshi54 — APPROVED** [review]
exported object and area, plus extra empty area. compiles and boots

**Comment by Lilaa3** [comment]
@jesusyoshi54 now that i think about it, would a separate pr for collision be fine?

**Comment by jesusyoshi54** [comment]
what would that PR be for?

**Comment by Lilaa3** [comment]
@jesusyoshi54 basically the same thing i guess

**Comment by jesusyoshi54** [comment]
I don't see a need for it tbh but I guess you can make it. Helpful for menu
type levels maybe. I would make it a new PR though if you want to do it.

**Pull request merged**